src/ca/bcit/bundles/lang_pt_BR.properties

```diff
@@ -0,0 +1,28 @@
+simulation_parameter_alpha=Alfa
+simulation_parameter_alpha_description=CHANGE THIS STRING
+simulation_parameter_alrorithm=Algoritmo
+simulation_parameter_alrorithm_description=CHANGE THIS STRING
```

Comment:
Link do opisu algorytmu

Reply:
This is the tooltip for the Algorithm parameter. Something like "The algorithm that will be ran"